pkg/acquisition/modules/docker/docker.go

```diff
@@ -409,6 +446,29 @@ func (d *DockerSource) EvalContainer(container dockerTypes.Container) (*Containe
 
 	}
 
+	if d.Config.UseContainerLabels {
+		parsedLabels := d.getContainerLabels(container.ID)
+		if len(parsedLabels) != 0 {
+			if v, ok := parsedLabels["enable"]; ok && strings.ToLower(v.(string)) == "true" {
+				if _, ok = parsedLabels["labels"]; !ok {
+					d.logger.Error("container has 'crowdsec.enable' label set to true but no 'labels' keys found")
+					return &ContainerConfig{}, false
+				}
+				d.logger.Debugf("container labels +%v", parsedLabels["labels"])
+				labels := make(map[string]string)
+				for k, v := range parsedLabels["labels"].(map[string]interface{}) {
```

Comment:
Same thing here if the user set `crowdsec.labels=foo` (which is a likely misconfiguration ?)